pkg/controller/priorityqueue/priorityqueue.go

```diff
@@ -57,9 +57,10 @@ func New[T comparable](name string, o ...Opt[T]) PriorityQueue[T] {
 	}
```

Comment:
[had to put this comment somewhere] I think this is the only remaining comment from my review that was not answered or adressed: https://github.com/kubernetes-sigs/controller-runtime/pull/3014#discussion_r1900795403

Reply:
Responded over there, it was needed to fix an import loop